server/server.js

```diff
@@ -117,26 +116,6 @@ const runServer = (
     res.status(200).send({ ip: req.ip })
   })
 
-  app.get('/cells/:filename', (req, res) => {
-    if (dataDir) {
-      return res.sendFile(
-        path.join(process.cwd(), dataDir, req.params.filename)
-      )
-    }
-    res.sendFile(path.join(__dirname, '../', 'data', req.params.filename))
-  })
-
```

Comment:
Hello @lilitkarapetyan :-)

In our current test implementation we're getting map tiles from the Internet. But, it's almost certain that future wargames will require locally hosted tiles - so we should keep the /tiles route, please.